apps/external_thread_pool_demo/micro_system.cc

```diff
@@ -0,0 +1,57 @@
+/**
+ * DAIL in Alibaba Group
+ *
+ */
+
+#include "micro_system.hh"
+#include "my_thread_pool.hh"
+
+void micro_system::register_pollers() {
+    _my_thread_pool_poller = seastar::reactor::poller(std::make_unique<my_thread_pool_pollfn>());
+}
+
+void micro_system::deregister_pollers() {
+    _my_thread_pool_poller = {};
+}
+
+seastar::future<> micro_system::stop() {
+    return seastar::smp::invoke_on_all([] {
+        micro_engine().deregister_pollers();
+    }).then([] {
+        my_thread_pool::stop();
+    });
+}
+
+seastar::future<> micro_system::configure() {
+    my_thread_pool::configure();
+    return seastar::smp::invoke_on_all([] {
+        allocate_micro_engine();
+        micro_engine().register_pollers();
+    });
+}
+
+void micro_system::exit() {
+    (void)stop().then([] {
```

Comment:
This is compensating for not calling stop() but it leaves behind stuff running in the background which is a dangerous bad practice.  You better just assert that stop was called (maybe have a bool _stopped member that's set by it), and let the caller be properly fixed.

Reply:
I got it